test/miscellaneous_tests/compound_macro.jl

```diff
@@ -60,36 +67,7 @@ let
     @test iscompound(rn.NH3_5)
 end
 
-### Test Various Independent Variables ###
```

Comment:
Again, moved down. You should see the pattern now. Realise this is a bit annoying to read, was intending to do this at the beginning of the summer (when everything else was sorted out). But as we wanted to go through and remake all the tests, I figured it was best to do this now to avoid two major reamkes of the test files.